buildscripts/condarecipe.local/meta.yaml

```diff
@@ -53,6 +53,9 @@ requirements:
     - tbb >=2021    # [not (armv6l or armv7l or aarch64 or linux32 or ppc64le)]
     # avoid confusion from openblas bugs
     - libopenblas !=0.3.6      # [x86_64]
+    # 0.3.17 buggy on M1 silicon
```

Comment:
Suggest adding a cross reference to the discussion on this as it demonstrates how the bugs materialise.